src/_includes/soft-mod-cheat-sheet-banner.html

```diff
@@ -2,8 +2,7 @@
 {% assign background_position = 'top left' %}
 {% capture title %}{% t software-modernisation.cheat-sheet-cta-banner-title %}{% endcapture %}
 {% capture description %}{% t software-modernisation.cheat-sheet-cta-banner-text %}{% endcapture %}
-{% capture cta_href %}
-      https://info.codurance.com/en/case-for-change-questionnaire
-{% endcapture %}
+{% capture cta_href %}{% t software-modernisation.cheat-sheet-cta-href %}{% endcapture %}
```

Comment:
If we're using a hubspot cta now, do we need these anymore?

Reply:
Will remove :)